packages/manager/modules/pci/src/projects/project/kubernetes/details/service/service.controller.js

```diff
@@ -25,6 +25,7 @@ export default class KubernetesServiceCtrl {
     this.CucCloudMessage = CucCloudMessage;
     this.CucControllerHelper = CucControllerHelper;
     this.Kubernetes = Kubernetes;
+    this.isKubernetesInstalling = true;
```

Comment:
**question:** Why do you need this? You're overriding the value from `resolve`.

Reply:
By default I wanted to disable the download option hence I set the value here.